p4include/pna.p4

```diff
@@ -373,7 +373,7 @@ enum PNA_CounterType_t {
 @noWarn("unused")
 extern Counter<W, S> {
   Counter(bit<32> n_counters, PNA_CounterType_t type);
-  void count(in S index);
```

Comment:
Has this been discussed in the smartNIC WG?

Reply:
No. I just created a PR for the same in pna repository.
https://github.com/p4lang/pna/pull/49

Reply:
So let's wait for that pr to merge first.